.github/workflows/ci.yml

```diff
@@ -898,14 +899,81 @@ jobs:
           cd semver
           rm output
           rm -f unexpectedly_did_not_fail
-
+      
```

Comment:
Assuming you use VSCode, here are a couple of useful settings to put in there:
```
    "files.trimFinalNewlines": true,
    "files.trimTrailingWhitespace": true,
    "files.insertFinalNewline": true,
```

Reply:
Yees, I use VSCode, thank you so much for these!